subprojects/robotpy-wpilib/gen/simulation/DCMotorSim.yml

```diff
@@ -19,6 +19,17 @@ classes:
       GetAngularAcceleration:
       GetTorque:
       GetInputVoltage:
-      Gearbox:
-      Gearing:
-      J:
+      GetGearbox:
+      GetGearing:
+      GetJ:
+
+inline_code: |
+  cls_DCMotorSim
```

Comment:
I just remembered, we added `inline_code` to classes for this use case didn't we?